chef-provisioning-aws.gemspec

```diff
@@ -17,7 +17,22 @@ Gem::Specification.new do |s|
 
   s.add_dependency "chef-provisioning", ">= 1.0", "< 3.0"
 
-  s.add_dependency "aws-sdk", [">= 2.2.18", "< 3.0"]
+  # all currently supported AWS services
+  s.add_dependency "aws-sdk-core", [">= 3.0", "< 4.0"]
+  s.add_dependency "aws-sdk-ec2", [">= 1.42.0", "< 2.0"]
+  s.add_dependency "aws-sdk-s3", [">= 1.17.0", "< 2.0"]
+  s.add_dependency "aws-sdk-rds", [">= 1.0", "< 4.0"]
+  s.add_dependency "aws-sdk-route53", [">= 1.0", "< 4.0"]
```

Comment:
The resource deps should all be ~> 1.0 since Amazon is on 1.x right now.

Reply:
Good catch - thanks for that!